docs/addRefreshAuthToDataProvider.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: default
+title: "addRefreshAuthToDataProvider"
+---
+
+# `addRefreshAuthToDataProvider`
+
+This helper function wraps an existing [`dataProvider`](./DataProviderIntroduction.md) to support authentication token refreshing mechanisms.
+
+## Usage
+
+Use `addRefreshAuthToDataProvider` to decorate an existing data provider. In addition to the base provider, this function takes a function responsible for refreshing the authentication token if needed.
+
+Here is a simple example that refreshes an expired JWT token when needed:
```

Comment:
I wonder if we should add this simple example as its own dedicated paragraph in the Security > Introduction page too. What do you think @fzaninotto ?

Reply:
👍

Reply:
Done